src/analysis/MOMA/MOMA.m

```diff
@@ -211,7 +211,7 @@
     % Solve the linearMOMA problem
     [QPproblem.A,QPproblem.b,QPproblem.F,QPproblem.c,QPproblem.lb,QPproblem.ub,QPproblem.csense,QPproblem.osense] = deal(A,b,F,c,lb,ub,csense,1);
     %QPsolution = solveCobraQP(QPproblem,[],verbFlag-1);
-    QPsolution = solveCobraQP(QPproblem, 'printLevel', verbFlag-1);
+    QPsolution = solveCobraQP(QPproblem, 'printLevel', verbFlag-1, 'method', 0);

```

Comment:
I don't like a Method overriding the settings made by a user, especially if this is "hidden" in the code.

Reply:
What do you propose instead?